src/parquet/file/reader.cc

```diff
@@ -160,16 +160,16 @@ void ParquetFileReader::DebugPrint(
     // Print column metadata
     for (auto i : selected_columns) {
       auto column_chunk = group_metadata->ColumnChunk(i);
-      const ColumnStatistics stats = column_chunk->statistics();
+      const auto stats = column_chunk->statistics();
```

Comment:
https://google.github.io/styleguide/cppguide.html#auto

aside: we should survey our usages of auto to make sure we are using `const auto&` in the right places